nginx/requirements.txt

```diff
@@ -0,0 +1 @@
+# integration pip requirements
```

Comment:
Let's keep track of this in a different issue yes 👍 

There are potentially a lot of edge cases but we probably don't want to solve all of them in a first iteration of the SDK